src/css/base.scss

```diff
@@ -2052,6 +2060,14 @@ textarea {
     height: 60px !important;
 }
 
+.h-80 {
+    height: 80px !important;
+}
+
+.h-300 {
```

Comment:
there is already 1 class with the same name

Reply:
Changed the other one to `mh-300` as it contains min-height prop.